common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1641,32 +1680,66 @@ object CelebornConf extends Logging {
     conf.getTimeAsMs("rss.partition.sort.timeout", "220s")
   }
 
-  def partitionSortMaxMemoryRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.partition.sort.memory.max.ratio", 0.1)
-  }
+  val PARTITION_SORTER_MAX_MEMORY_RATIO: ConfigEntry[Double] =
+    buildConf("celeborn.partition.sorter.memory.maxRatio")
+      .withAlternative("rss.partition.sort.memory.max.ratio")
+      .categories("worker")
+      .doc("Max ratio of sort memory.")
+      .version("0.2.0")
+      .doubleConf
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0].")
+      .createWithDefault(0.1)
 
-  def workerPausePushDataRatio(conf: CelebornConf): Double = {
-    conf.getDouble("rss.pause.pushdata.memory.ratio", 0.85)
-  }
+  val WORKER_PAUSE_PUSH_DATA_RATIO: ConfigEntry[Double] =
+    buildConf("celeborn.worker.memory.pause.pushData.ratio")
```

Comment:
`celeborn.worker.pausePushData.memory.ratio` @pan3793

Reply:
the doc says
> If direct memory usage reach this limit ...

Is it dedicated to direct memory? If yes, should we mention it in the name?